fe/fe-core/src/main/java/com/starrocks/analysis/CreateFunctionStmt.java

```diff
@@ -138,6 +178,27 @@ private void analyzeCommon(Analyzer analyzer) throws AnalysisException {
         if (md5sum != null && !md5sum.equalsIgnoreCase(checksum)) {
             throw new AnalysisException("library's checksum is not equal with input, checksum=" + checksum);
         }
+
+        if (isStarrocksJar) {
+            analyzeUdfClassInStarrocksJar();
+        }
+    }
+
+    private void analyzeUdfClassInStarrocksJar() throws AnalysisException {
+        String symbol = properties.get(SYMBOL_KEY);
+        if (Strings.isNullOrEmpty(symbol)) {
```

Comment:
why don't use `containsKey`? The value of the map is nullable?

Reply:
No, becuase we are going to use `symbol` later.